### DISCUSSION TITLE
Migration statuses

### DISCUSSION THREAD

**Discussion by CarlosLeonCode** [jeremyevans/sequel#1914] [General]
Hey guys!
 
There is some way to check my migrations statuses? as the **db:migrate:status** command in a Rails app.
I used the below methods:
- **Sequel::Migrator.is_current?**
- **Sequel::Migrator.check_current**

But none of them works for me.

I got the following error.
![image](https://user-images.githubusercontent.com/40130036/186575900-d4ceed70-c210-4ce8-b85f-c301a7fe8c6f.png)

And I just have two migrations:
![image](https://user-images.githubusercontent.com/40130036/186575959-34ae3a6f-aa5d-49cc-84e0-1b480978604c.png)
![image](https://user-images.githubusercontent.com/40130036/186575995-dff4ef64-874a-432b-a6fb-be6a65aae611.png)

There are inside **db/migrations** path:
![image](https://user-images.githubusercontent.com/40130036/186576090-5ff3bcec-ad9c-4339-a132-e63589a36ac4.png)

My Rakefile looks like:
![image](https://user-images.githubusercontent.com/40130036/186576166-3d06595d-057e-4f8b-9997-ff99f22f8bb3.png)

And the database structure is:
![image](https://user-images.githubusercontent.com/40130036/186576285-b03687c3-62fb-48b7-ad53-99a8bf2b6f3c.png)

I don't get it why I got that error when I used one of that commands to check my migration statuses. I suppose that I have to create a task to emulate the **db:migrate:status** command using one of previous commands.

This is the [Repository](https://github.com/CarlosLeonCode/lrn-sequel-ruby) with the complete code if someone wants to check it out :smiley:

**Comment by AlexWayfer** [comment]
As I remember, you should run migrations once to auto-create Sequel technical tables.

**Comment by janko** [comment]
It seems like you're passing the database URL into the mentioned method for checking migration status instead of a Sequel database object.

**Reply by AlexWayfer** [reply]
> It seems like you're passing the database URL into the mentioned method for checking migration status instead of a Sequel database object.

Oh, the error is about method name, not table name. My bad, you're right.

How to create a connection (and `DB` object): https://sequel.jeremyevans.net/rdoc/files/doc/opening_databases_rdoc.html

**Reply by CarlosLeonCode** [reply]
Ohh! I'm going to check that, ty!

**Reply by CarlosLeonCode** [reply]
Effectively that was my issue. The solution is to pass a DB obj connection. Thank you again guys @AlexWayfer & @janko 
Regards!
![image](https://user-images.githubusercontent.com/40130036/186976197-28a5cab2-b976-48c7-9464-589fe4f5094c.png)
